src/microbe_stage/Microbe.cs

```diff
@@ -789,21 +789,55 @@ public void Divide()
         copyEntity.Compounds.ClearCompounds();
 
         var keys = new List<Compound>(Compounds.Compounds.Keys);
+        var reproductionCompounds = CalculateTotalCompounds();
 
-        // Split the compounds evenly between the two cells.
+        // Split the compounds between the two cells.
         foreach (var compound in keys)
         {
             var amount = Compounds.GetCompoundAmount(compound);
 
             if (amount > 0)
             {
-                Compounds.TakeCompound(compound, amount * 0.5f);
+                float divideAmount;
```

Comment:
This can be inlined in the out specification

Reply:
That one I definitely didn't know, hahah.